apps/wallet-dashboard/components/Dialogs/Staking/views/UnstakeView.tsx

```diff
@@ -26,22 +26,25 @@ import {
 } from '@iota/core';
 import { IOTA_TYPE_ARG } from '@iota/iota-sdk/utils';
 import { useMemo } from 'react';
-import { useCurrentAccount, useSignAndExecuteTransaction } from '@iota/dapp-kit';
+import { useCurrentAccount } from '@iota/dapp-kit';
 import { Loader, Warning } from '@iota/ui-icons';
 import { useUnstakeTransaction } from '@/hooks';
 import { ValidatorStakingData } from '@/components';
 import { DialogLayout, DialogLayoutFooter, DialogLayoutBody } from '../../layout';
+import { Transaction } from '@iota/iota-sdk/transactions';
 
 interface UnstakeDialogProps {
```

Comment:
Mind renaming this while you are it?
```suggestion
interface UnstakeViewProps {
```